src/http/lib/pubsub.ts

```diff
@@ -1,6 +1,6 @@
 import fetch from 'node-fetch';
 import WebServerSingleton from "./WebServer/WebServerSingleton";
-import { TypeDaprPubSub } from '../types/DaprPubSub.type';
+import { TypeDaprPubSub, TypeElementOfDaprPubSub } from '../types/DaprPubSub.type';
```

Comment:
I would name this: TypeDaprPubSubSubscriptions

Reply:
but it is just for one element of the array. so I think that's okay. if you want I will change it later